web/src/routes/backOffice/containers/details/Container.svelte

```diff
@@ -35,6 +42,36 @@
 		return res.data;
 	}
 
+	/**
+	 * Deletes the container displayed on the page.
+	 */
+	async function deleteContainer() {
+		const res = await ecomapHttpClient.DELETE("/containers/{containerId}", {
+			params: {
+				path: {
+					containerId: id,
+				},
+			},
+		});
+
+		if (res.error) {
```

Comment:
In case of a 409 response, we probably want to display a custom message.
Swagger reference: `Container is associated with a report, bookmark, or route.`